internal/v1/server.go

```diff
@@ -286,3 +286,8 @@ func (s *Server) getDistro(ctx echo.Context, distro Distributions) (*distributio
 	}
 	return d, nil
 }
+
+// used to access DB directly in tests
+func getDB(s *Server) db.DB {
```

Comment:
Nitpick: [Avoid getters/setters](https://go.dev/doc/effective_go#Getters).

Reply:
Right, but this is just to have a function to export to the test package. Alternatively i'll just make the db on the server struct public.

Reply:
found a nicer way